From 10e0beec47def7813e2e256697018f3d0a4559fe Mon Sep 17 00:00:00 2001
From: Derek Foreman <derekf@osg.samsung.com>
Date: Fri, 6 Nov 2015 13:15:35 -0300
Subject: [PATCH 1/1] ARM: bcm2709_defconfig: Enable config options for vc4
 support

Add the needed Kconfig symbols to build Eric Anholt's WIP kernel
4.1 Linux kernel with vc4 dri/kms/3d support. Also increase CMA
size from 5 MiB to 256 MiB as that is needed by the driver.

Signed-off-by: Derek Foreman <derekf@osg.samsung.com>
---
 arch/arm/configs/bcm2709_defconfig | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index c2ddd10dc106..4312906bfe0f 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -394,7 +394,7 @@ CONFIG_NFC_PN533=m
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 CONFIG_DMA_CMA=y
-CONFIG_CMA_SIZE_MBYTES=5
+CONFIG_CMA_SIZE_MBYTES=256
 CONFIG_MTD=m
 CONFIG_MTD_BLOCK=m
 CONFIG_MTD_NAND=m
@@ -807,6 +807,8 @@ CONFIG_FB=y
 CONFIG_FB_BCM2708=y
 CONFIG_FB_SSD1307=m
 CONFIG_FB_RPISENSE=m
+CONFIG_DRM=y
+CONFIG_DRM_VC4=y
 # CONFIG_BACKLIGHT_GENERIC is not set
 CONFIG_BACKLIGHT_GPIO=m
 CONFIG_FRAMEBUFFER_CONSOLE=y
-- 
2.4.3

